source/streaming-mode/streaming-read.txt

```diff
@@ -0,0 +1,382 @@
+.. _streaming-read-from-mongodb: 
+
+===================================
+Read from MongoDB in Streaming Mode
+===================================
+
+.. toctree::
+   :caption: Streaming Read Configuration Options
+
+   /streaming-mode/streaming-read-config
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol 
+
+Overview
+--------
+
+When reading a stream from a MongoDB database, the {+connector-long+} supports both 
+*micro-batch processing* and 
+*continuous processing*. Micro-batch processing is the default processing engine, while
+continuous processing is an experimental feature introduced in 
+Spark version 2.3. To learn 
+more about continuous processing, see the `Spark documentation <https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html#continuous-processing>`__.
+
+.. include:: /includes/fact-read-from-change-stream
+
+.. tabs-drivers::
+
+   tabs:
+
+     - id: java-sync
```

Comment:
Is there a reason the content in these tabs aren't separated into includes files like the batch files are? I think it's fine to keep for this PR, but might be useful to have a ticket for sometime down the line to split these out

Reply:
agreed. it was like this when i got here.